apps/desktop/src/components/CommitMessageEditor.svelte

```diff
@@ -165,17 +166,31 @@
 		onchange={(value) => {
 			onChange?.({ title: value });
 		}}
+		oncompositionstart={() => {
+			// Track IME composition state manually for WebKit compatibility
+			isTitleComposing = true;
+		}}
 		onkeydown={async (e: KeyboardEvent) => {
+			// Prevent focus movement when Enter is pressed during IME composition
+			// for Japanese/Chinese/Korean input confirmation.
+			// WebKit sets e.isComposing to false on Enter, so we use isTitleComposing as fallback
+			if (e.key === 'Enter' && (e.isComposing || isTitleComposing)) {
```

Comment:
Do we need to check `e.isComposing` when we have `isTitleComposing` from the `oncompositionstart` event?